### PR TITLE
feat: Aerial Wasm contracts migration

### PR DESCRIPTION
**Pull request by MissingNO57**
- Added migration command for Aerial contract API.
- Migration schema support
- Added fix for rest-api
- Updated simple.wasm to support migration

### CODE REVIEW

**Review by solarw — APPROVED** [review]
lgtm

File: cosmpy/aerial/contract/__init__.py [hunk 1]
```diff
@@ -235,6 +236,69 @@ def instantiate(
...
+
+        :return: contract address
+        """
+        assert self._address, RuntimeError("Address was not set.")
+
+        if self._migrate_schema is not None:
+            validate(args, self._migrate_schema)
+
+        # build up the migrate transaction
+        instatiate_msg = create_cosmwasm_migrate_msg(
```
Comment (jrriehl) [inline cosmpy/aerial/contract/__init__.py:289]: Should this be `migrate_msg`?

File: tests/integration/test_contract.py [hunk 2]
```diff
@@ -116,6 +118,10 @@ def test_deployed_contract(self):
         assert result["exists"]
         assert result["value"] == value
 
+        # Upgrade contract
+        tx_res = deployed_contract.upgrade({}, wallet, CONTRACT_PATH)
+        assert tx_res.response
```
Comment (jrriehl) [inline tests/integration/test_contract.py:123]: It's not obvious to me that this means the upgrade was successful. Maybe we can check something like that the `code_id` changed and the `address` stayed the same?

**Review by jrriehl — COMMENTED** [review]
Nice one! Just a couple of small suggestions.

**Review by 5A11 — APPROVED** [review]

**Pull request merged**